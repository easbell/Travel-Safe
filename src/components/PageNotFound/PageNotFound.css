.page-not-found {
  text-align: center;
  font-family: Montserrat, sans-serif;
  font-weight: 400;
  margin: 0
}

.not-found {
  font-size: 150px;
  margin: 0;
  font-weight: 600;
}

.link {
  text-decoration: none;
  color: #000;
  &:hover {
    color: #838282;
  }
}