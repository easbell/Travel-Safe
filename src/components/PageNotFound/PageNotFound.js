import React from 'react';
import { Link } from 'react-router-dom';
import './PageNotFound.css';

export const PageNotFound = () => {
	return (
		<div className='page-not-found'>
			<h2>This is not the page</h2>
			<h2>you are looking for.</h2>
			<h2 className='not-found'>404</h2>
		</div>
	)
}