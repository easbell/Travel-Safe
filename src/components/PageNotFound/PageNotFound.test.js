import React from 'react';
import { PageNotFound } from './PageNotFound';
import { shallow } from 'enzyme';

describe('PageNotFound', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
      <PageNotFound />)
  });

  it('should match snapshot', () => {
      expect(wrapper).toMatchSnapshot();
  });
});