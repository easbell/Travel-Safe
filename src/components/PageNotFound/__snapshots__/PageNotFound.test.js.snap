// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound should match snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PageNotFound />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h2>
          This is not the page
        </h2>,
        <h2>
          you are looking for.
        </h2>,
        <h2
          className="not-found"
        >
          404
        </h2>,
      ],
      "className": "page-not-found",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "This is not the page",
        },
        "ref": null,
        "rendered": "This is not the page",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "you are looking for.",
        },
        "ref": null,
        "rendered": "you are looking for.",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "404",
          "className": "not-found",
        },
        "ref": null,
        "rendered": "404",
        "type": "h2",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h2>
            This is not the page
          </h2>,
          <h2>
            you are looking for.
          </h2>,
          <h2
            className="not-found"
          >
            404
          </h2>,
        ],
        "className": "page-not-found",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "This is not the page",
          },
          "ref": null,
          "rendered": "This is not the page",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "you are looking for.",
          },
          "ref": null,
          "rendered": "you are looking for.",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "404",
            "className": "not-found",
          },
          "ref": null,
          "rendered": "404",
          "type": "h2",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
